### ISSUE TITLE
please provide a warning that the rexel energeasy connect is not supported anymore

### ISSUE THREAD

**Issue by jeremyVignelles** [home-assistant/home-assistant.github.io#33763]
### Feedback

See https://github.com/iMicknl/ha-tahoma/issues/591
Rexel has made a breaking change and has no plan to fix it

### URL

https://www.home-assistant.io/integrations/rexel/

### Version

2024.7.2

**Comment by jeremyVignelles** [comment]
👍

**Comment by jeremyVignelles** [comment]
Still an issue

**Comment by Pouet71** [comment]
Hello, yes still an issue. I just past one hour to connect my energeasy found by overkiz...
My login nerver works...
Thanks for the whole works !

**Comment by iMicknl** [comment]
Energeasy Connect is not mentioned on the https://www.home-assistant.io/integrations/overkiz/ anymore.

**Comment by jeremyVignelles** [comment]
@iMicknl : Yes, but that page still exists and states that it should work with overkiz : https://www.home-assistant.io/integrations/rexel/
I don't know the details, but it would be nice to add a warning there that it should work up to a specific version or something.
